Rakefile

```diff
@@ -69,5 +80,42 @@ namespace :brew do
     cask_erb = ERB.new(File.read(File.join(__dir__, 'templates', "#{cask}.rb.erb")), 0, '-')
     File.write(File.join(__dir__, 'Casks', "#{cask}.rb"), cask_erb.result(binding))
   end
-end
 
+  desc 'Update SHAs for pe-client-tools: rake brew:pe_client_tools[19.3.0] or rake brew:pe_client_tools[19.3.0,2019.3]'
+  task :pe_client_tools, [:version, :collection] do |task, args|
+    pkg = 'pe-client-tools'
+    version = args[:version]
+    collection = args[:collection] || LATEST_PE
+    cask = pkg
+    cask += '-' + args[:collection] if args[:collection]
+    os_versions = operating_systems("pct#{collection}")
+
+    host = 'pm.puppet.com'
+    path_pre = "/pe-client-tools/#{collection}.0/"
+
+    package_triples = Net::HTTP.start(host, use_ssl: true) do |http|
+      latest_versions = os_versions.map do |os_ver|
+        version
```

Comment:
This is making latest_versions `[nil,nil]` which is why you are getting the sha error.

Reply:
That results in 
```diff
--- a/Casks/pe-client-tools.rb
+++ b/Casks/pe-client-tools.rb
@@ -2,12 +2,12 @@ cask 'pe-client-tools' do
   case MacOS.version
   when '10.12'
     os_ver = '10.12'
-    version '19.3.0'
-    sha256 'b1c84178f8492a69ffc7d75644d365b3933d5eb1501f7fa84c54ea6237b820f2'
+    version '10.12'
+    sha256 '29b1f9a18fafb4444734ac5a901cd544ef2f91bf5ac7156a5d3bdd73413e7a90'
   else
     os_ver = '10.13'
-    version '19.3.0'
-    sha256 'c40be414914be42f69555da4d3997703dad21ea7b05883cdb58e3c72e4c549f1'
+    version '10.13'
+    sha256 '2904ef837e347076939a19968ed11eed87231434a904554ecfe4982087923cc8'
   end
```
Which I think the template might need some tweeking.

Reply:
Oh, NM we have required params to this rake task. The shas are just different bc the body is a http 302.